trio/tests/test_highlevel_ssl_helpers.py

```diff
@@ -40,7 +40,10 @@ class FakeHostnameResolver(trio.abc.HostnameResolver):
 
 
 # This uses serve_ssl_over_tcp, which uses open_ssl_over_tcp_listeners...
-async def test_open_ssl_over_tcp_stream_and_everything_else():
+# noqa is needed because flake8 doesn't understand how pytest fixtures work.
```

Comment:
I think you can fix the flake8 issue by removing the unused client_ctx import

Reply:
It's not unused, it's making the fixture function visible to pytest...

Reply:
Sorry about that, I'm used to projects where pytest discovers the fixtures through contest.py